NSS-Likeable-Lemurs/BangazonSite
# Second Part Of 32

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/32)

## Description of Proposed Changes
- Altered _LoginPartial and _Layout files so that the "sell a product" would only appear after a user has logged in and authorized.
- "Sell A Product" will take you to the create section of the Product view.
- Added "city" and "image path" to applicationDBcontext file so that the city and image path fields in the "create" product form would appear as required fields.
- If not all fields are completed, they will be notified upon click of the "sell" button

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <secondPartOf32>
Launch the application and log in. Click on "sell a product" and enter some information. Make sure to check each field so that the error message will appear after clicking on the "sell" button.
```


## Impacted Areas in Application
_Layout
_LoginPartial
Products - Create.cshtml
ProductTypesController

List general components of the application that this PR will affect:

* Products/ProductTypesController

## Mentions @kellycook301 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
